cloud/amazon/ec2_group_facts.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This Ansible library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: ec2_group_facts
+short_description: Gather facts about security groups in AWS
+description:
+    - Gather facts about security groups in AWS
+version_added: "2.2"
+author: "Destry Jaimes (@calamityman)"
+options:
+  filters:
+    description:
+      - A dict of filters to apply. Each dict item consists of a filter key and a filter value. See U(http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSecurityGroups.html) for possible filters.
+    required: false
+    default: null
+    
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+
+# Gather facts about all security groups
+- ec2_group_facts:
+
+# Gather facts about security groups in a particular VPC using VPC ID
+- ec2_group_facts:
+    filters:
+      vpc-id: vpc-12345678
+
+# Gather facts about security groups in a particular VPC using security group name
+- ec2_group_facts:
+    filters:
+      group-name: "default"
```

Comment:
Can you do this small change and addition, please? I think it then becomes clearer that we have alternatives:

```
# Gather facts about security groups in a particular VPC using security group name
- ec2_group_facts:
    filters:
      group-name: example

# Gather facts about multiple security groups in a particular VPC using security group names
- ec2_group_facts:
    filters:
      group-name:
        - example-1
        - example-2
        - example-3